pycheribuild/projects/cross/opensbi.py

```diff
@@ -213,19 +216,42 @@ def run_tests(self):
 class BuildAllianceOpenSBI(BuildOpenSBI):
     target = "cheri-alliance-opensbi"
     _default_install_dir_fn = ComputedDefaultValue(
-        function=lambda config, p: config.cheri_sdk_dir / "cheri-alliance-opensbi/riscv64",
-        as_string="$SDK_ROOT/cheri-alliance-opensbi/riscv64",
+        function=lambda config, p: config.cheri_alliance_sdk_dir / "cheri-alliance-opensbi/riscv64",
```

Comment:
```suggestion
        function=lambda config, p: config.cheri_alliance_sdk_dir / "opensbi/riscv64",
```
probably don't need a double prefix here?

Reply:
Yeah, good point.